content/docs/rfcs/38/README.md

```diff
@@ -0,0 +1,507 @@
+---
+slug: 38
+title: 38/LOGOS-CONSENSUS
+name: Logos Glacier Consensus Protocol
+status: raw
+category: informative
+tags: logos/consensus,implementation/rust, implementation/python, implementation/common-lisp
+editor: Mark Evenson <mark.evenson@status.im>
+created: 01-JUL-2022
+revised: <2022-08-10 Wed 10:15Z>
+contributors:
+    - Álvaro Castro-Castilla 
+---
+
+# Abstract
+
+We propose to transition our use of resource intensive Nakomoto
+consensus mechanisms to the class of more efficient, secure,
+leaderless ones rooted in decentralization.  We sketch a simple
+two-level model for the practical execution of such a fairly
+distributed consensus mechanism, in which an underlying leaderless
+binary decision mechanism is utilized to used to vote on the
+construction of a distributed, directed, acyclic graph.  We present
+the Glacier algorithm which provides a Byzantine fault tolerant
+implementation of the base binary decision mechanism.  We outline a
+taxonomy of Byzantine adversaries that seek to thwart this
+computation's correct honesty.
+
```

Comment:
Requirements would fit nicely here

Reply:
Ditto something like "prior work" or "motivation" (comparing e.g. BTC/ETH consensus briefly)

Reply:
I'll make a stab at further explaining the prior work of leaderless BFT binary consensus.